Origin: upstream https://github.com/varnishcache/varnish-cache/commit/785ecf0a8fd5f0d4864cf9091379f463f950d373
Description: Bump libvarnishapi soname version from 1.0.6 to 2.0.0
From 785ecf0a8fd5f0d4864cf9091379f463f950d373 Mon Sep 17 00:00:00 2001
From: Denes Matetelki <dmatetelki@varnish-software.com>
Date: Thu, 21 Jun 2018 13:22:56 +0200
Subject: [PATCH] Increasing libvarnishapi.so version from 1.0.6 to 2.0.0

Fixes: libvarnishapi needs soname bump #2710

Should be backported to branches: 5.0, 5.1, 5.2, 6.0
---
 lib/libvarnishapi/Makefile.am | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/lib/libvarnishapi/Makefile.am b/lib/libvarnishapi/Makefile.am
index e679de3acf..5e13de019c 100644
--- a/lib/libvarnishapi/Makefile.am
+++ b/lib/libvarnishapi/Makefile.am
@@ -9,7 +9,7 @@ AM_CPPFLAGS = \
 
 lib_LTLIBRARIES = libvarnishapi.la
 
-libvarnishapi_la_LDFLAGS = $(AM_LDFLAGS) -version-info 1:6:0
+libvarnishapi_la_LDFLAGS = $(AM_LDFLAGS) -version-info 2:0:0
 
 libvarnishapi_la_SOURCES = \
 	vsl_api.h \
